src/modules/ekf2/ekf2_params.c

```diff
@@ -521,6 +531,18 @@ PARAM_DEFINE_FLOAT(EKF2_GPS_V_GATE, 5.0f);
 PARAM_DEFINE_FLOAT(EKF2_TAS_GATE, 3.0f);
 
 /**
+ * Replay mode
+ *
+ * A value of 1 indicates that the ekf2 module will publish
+ * replay messages for logging.
+ *
+ * @group EKF2
+ * @reboot_required true
+ * @boolean
+ */
+PARAM_DEFINE_INT32(EKF2_REC_RPL, 0);
```

Comment:
Something I don't understand has happened to this parameter because there is no longer any code that uses it. I need to investigate that further.

Reply:
EKF2 replay of sdlog2 data is no longer supported, so the EKF2_REC_RPL parameter is no longer required. All logging is control inside the logger app which logs the ekf2_timestamps data required for the system logger to run an ekf2 replay.

I will push a patch which removes this parameter.

Reply:
EKF2_REC_RPL was deleted and is not in master.